docs-gastby/src/pages/getting-started/index.mdx

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
This file (and `getting-started` directory) can be removed, since we don't want to publish template pages that are not relevant to the UI.